product_warranty/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Product warranty
+================
+
+This module allows the user to specify a warranty on a product.
+
+The warranty can then be used in other modules.
+
+Usage
+=====
+
+* Go to a Product
+* Enter the warranty
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/product-attribute/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Miguel Ángel Gómez Trillo <BT-mgomez@braintec-group.com>
+
+Funders
+-------
+
+The development of this module has been financially supported by:
+
+* Open Source Integrators <http://www.opensourceintegrators.com>
```

Comment:
Here you should put BT and perhaps xplace. What do you think @BT-mfasola ?

Reply:
Done.